allennlp/service/config_explorer.py

```diff
@@ -0,0 +1,204 @@
+"""
+config explorer
+"""
+from typing import List, Union, Optional, Dict, Sequence, Tuple
+import argparse
+import logging
+from string import Template
+import re
+import sys
+
+from flask import Flask, request, Response, jsonify
+from flask_cors import CORS
+from gevent.pywsgi import WSGIServer
+
+from allennlp.common.configuration import configure, Config, ConfigItem, full_name, _NO_DEFAULT
+from allennlp.common.util import import_submodules
+from allennlp.service.server_flask import ServerError
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+def make_app(include_packages: Sequence[str] = ()) -> Flask:
+    """
+    Creates a Flask app that serves up a simple configuration wizard.
+    """
+    # Load modules
+    for package_name in include_packages:
+        import_submodules(package_name)
+
+    app = Flask(__name__)  # pylint: disable=invalid-name
+
+    @app.errorhandler(ServerError)
+    def handle_invalid_usage(error: ServerError) -> Response:  # pylint: disable=unused-variable
+        response = jsonify(error.to_dict())
+        response.status_code = error.status_code
+        return response
+
+    @app.route('/')
+    def index() -> Response: # pylint: disable=unused-variable
+        class_name = request.args.get('class', '')
+
+        try:
+            config = configure(class_name)
+        except:
+            # TODO(joelgrus): better error handling
```

Comment:
Just pointing out the TODO.  I assume you meant to leave it but wanted to make sure.